src/main/flight/pid.h

```diff
@@ -237,6 +237,9 @@ typedef struct pidProfile_s {
     uint8_t angle_feedforward_smoothing_ms; // Smoothing factor for angle feedforward as time constant in milliseconds
     uint8_t angle_earth_ref;         // Control amount of "co-ordination" from yaw into roll while pitched forward in angle mode
     uint16_t horizon_delay_ms;           // delay when Horizon Strength increases, 50 = 500ms time constant
+    uint8_t tpa_rate_lower;                 // Percent reduction in P or D at zero throttle
```

Comment:
Please ensure the comment is absolutely clear.  For example, does a value of 20 mean that the TPA effect will cut D to 20% of normal, or result in D that is 80% normal, at zero throttle?
Currently, TPA high of 80 will cut the D to 20% of normal.

Reply:
hey chris, since tpa lower is just an extension of tpa I basically copied the logic of how it is applied and adjusted the description according to tpa_rate and tpa_breakpoint. default values in the table above are now correct and the red line in the figure represents the defaults. maybe @SupaflyFPV has some clearer explanations?

Reply:
Pichi is very clear about in his comment here. "Percent reduction" means that you will get 20% attenuation/reduction, or 80% of your original value.

Reply:
yeah its the same as TPA_RATE. It follows the same logic, just for the LOWER breakpoint. 👌